### ISSUE TITLE
[Component] [radio-group, radio-button] el-radio-button @change事件错误

### ISSUE THREAD

**Issue by maxzhou0** [element-plus/element-plus#12043]
Bug Type: **`Component`**

## Environment

- Vue Version: `3.2.47`
- Element Plus Version: `2.3.0`
- Browser / OS: `Chrome`
- Build Tool: `Vite`

### Related Component

- `el-radio-group`
- `el-radio-button`

### Reproduction Link

[Element Plus Playground](https://element-plus.run/#eyJBcHAudnVlIjoiPHRlbXBsYXRlPlxuICA8ZGl2PlxuICAgIDxlbC1yYWRpby1ncm91cCA6bW9kZWwtdmFsdWU9XCJyYWRpbzFcIiBAY2hhbmdlPVwib25DaGFuZ2VcIiBzaXplPVwibGFyZ2VcIj5cbiAgICAgIDxlbC1yYWRpby1idXR0b24gbGFiZWw9XCJOZXcgWW9ya1wiIC8+XG4gICAgICA8ZWwtcmFkaW8tYnV0dG9uIGxhYmVsPVwiV2FzaGluZ3RvblwiIC8+XG4gICAgICA8ZWwtcmFkaW8tYnV0dG9uIGxhYmVsPVwiTG9zIEFuZ2VsZXNcIiAvPlxuICAgICAgPGVsLXJhZGlvLWJ1dHRvbiBsYWJlbD1cIkNoaWNhZ29cIiAvPlxuICAgIDwvZWwtcmFkaW8tZ3JvdXA+XG4gIDwvZGl2PlxuICA8ZGl2IHN0eWxlPVwibWFyZ2luLXRvcDogMjBweFwiPlxuICAgIDxlbC1yYWRpby1ncm91cCB2LW1vZGVsPVwicmFkaW8yXCI+XG4gICAgICA8ZWwtcmFkaW8tYnV0dG9uIGxhYmVsPVwiTmV3IFlvcmtcIiAvPlxuICAgICAgPGVsLXJhZGlvLWJ1dHRvbiBsYWJlbD1cIldhc2hpbmd0b25cIiAvPlxuICAgICAgPGVsLXJhZGlvLWJ1dHRvbiBsYWJlbD1cIkxvcyBBbmdlbGVzXCIgLz5cbiAgICAgIDxlbC1yYWRpby1idXR0b24gbGFiZWw9XCJDaGljYWdvXCIgLz5cbiAgICA8L2VsLXJhZGlvLWdyb3VwPlxuICA8L2Rpdj5cbiAgPGRpdiBzdHlsZT1cIm1hcmdpbi10b3A6IDIwcHhcIj5cbiAgICA8ZWwtcmFkaW8tZ3JvdXAgdi1tb2RlbD1cInJhZGlvM1wiIHNpemU9XCJzbWFsbFwiPlxuICAgICAgPGVsLXJhZGlvLWJ1dHRvbiBsYWJlbD1cIk5ldyBZb3JrXCIgLz5cbiAgICAgIDxlbC1yYWRpby1idXR0b24gbGFiZWw9XCJXYXNoaW5ndG9uXCIgZGlzYWJsZWQgLz5cbiAgICAgIDxlbC1yYWRpby1idXR0b24gbGFiZWw9XCJMb3MgQW5nZWxlc1wiIC8+XG4gICAgICA8ZWwtcmFkaW8tYnV0dG9uIGxhYmVsPVwiQ2hpY2Fnb1wiIC8+XG4gICAgPC9lbC1yYWRpby1ncm91cD5cbiAgPC9kaXY+XG48L3RlbXBsYXRlPlxuXG48c2NyaXB0IGxhbmc9XCJ0c1wiIHNldHVwPlxuaW1wb3J0IHsgcmVmIH0gZnJvbSAndnVlJ1xuY29uc3Qgb25DaGFuZ2UgPSAodmFsOmFueSk9PntcbiAgcmFkaW8xLnZhdWUgPSB2YWxcbn1cbmNvbnN0IHJhZGlvMSA9IHJlZignTmV3IFlvcmsnKVxuY29uc3QgcmFkaW8yID0gcmVmKCdOZXcgWW9yaycpXG5jb25zdCByYWRpbzMgPSByZWYoJ05ldyBZb3JrJylcbjwvc2NyaXB0PlxuIiwiaW1wb3J0X21hcC5qc29uIjoie1xuICBcImltcG9ydHNcIjoge31cbn0iLCJfbyI6e319)

### Steps to reproduce

如果el-radio-group 是用:model-value 和 @change事件做控制，且值用el-radio-button显示时，点击选项，

### What is Expected?

正确显示

### What is actually happening?

显示会不正确

### Additional comments

_(empty)_

**Comment by MrWeilian** [comment]
你的 ref 取 value 拼写错了。
![image](https://user-images.githubusercontent.com/30046649/225225734-48248576-b09b-402b-826f-311b5147b72c.png)

**Comment by maxzhou0** [comment]
> 你的 `ref` 取 `value` 拼写错了。改正试过了没啥问题 ![image](https://user-images.githubusercontent.com/30046649/225225734-48248576-b09b-402b-826f-311b5147b72c.png)

抱歉，我修改了代码，但又有新的错误。
如果在@change事件中不对value进行赋值，界面应该维持原来的选择。
但实际上视图就无法回到正常状态了。麻烦看一下。

![1678876203336](https://user-images.githubusercontent.com/18253226/225283189-a041ed00-098c-42b5-aec5-db51da105703.png)

el-radio-button有这个问题，el-radio没有。

**Comment by ixifree** [comment]
> > 你的 `ref` 取 `value` 拼写错了。改正试过了没啥问题 ![image](https://user-images.githubusercontent.com/30046649/225225734-48248576-b09b-402b-826f-311b5147b72c.png)
> 
> 抱歉，我修改了代码，但又有新的错误。 如果在@change事件中不对value进行赋值，界面应该维持原来的选择。 但实际上视图就无法回到正常状态了。麻烦看一下。
> 
> ![1678876203336](https://user-images.githubusercontent.com/18253226/225283189-a041ed00-098c-42b5-aec5-db51da105703.png)
> 
> el-radio-button有这个问题，el-radio没有。

你的问题和我提的#12046是类似的，#12050已经修复了这个问题

**Comment by acyza** [comment]
或许`:modelValue`应该搭配`update:modelValue`事件使用，如果只是`modelValue`相当于绑定值不可改变，根据[文档](https://element-plus.gitee.io/zh-CN/component/radio.html#radiogroup-events)的描述个人感觉绑定值不改变不该触发`change`事件。

**Comment by Dsaquel** [comment]
>或许:modelValue应该搭配update:modelValue事件使用，如果只是:modelValue相当于绑定值不可改变，根据[文档](https://element-plus.gitee.io/zh-CN/component/radio.html#radiogroup-events)的描述个人感觉绑定值不改变不该触发change事件。

Agreed with this statement and it is the default radio html change event.
Closing since this is not an issue.
